// @test error

#foo

fn main() : int
{
	return 0;
}